spec/features/decks_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+require "support/login"
+require 'database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.